paymentsheet/src/test/java/com/stripe/android/paymentelement/embedded/content/DefaultEmbeddedConfigurationCoordinatorTest.kt

```diff
@@ -178,6 +181,33 @@ internal class DefaultEmbeddedConfigurationCoordinatorTest {
         }
     }
 
+    @Test
+    fun `configure correctly parses appearance`() = testScenario {
+        val configuration = EmbeddedPaymentElement.Configuration
+            .Builder("Example, Inc.")
+            .appearance(
+                PaymentSheet.Appearance(
+                    colorsLight = PaymentSheetAppearance.CrazyAppearance.appearance.colorsLight,
+                )
+            ).build()
+        configurationHandler.emit(Result.success(createPaymentElementLoaderState()))
+        configurationCoordinator.configure(
+            intentConfiguration = PaymentSheet.IntentConfiguration(
+                PaymentSheet.IntentConfiguration.Mode.Payment(5000, "USD"),
+            ),
+            configuration = configuration
+        )
+
+        assertThat(StripeTheme.colorsLightMutable.componentBorder)
```

Comment:
Seems like we should have an assert like this before we call configure.

We should also reset after the test is over, if possible.